pkg/federation/sdlmerge/sdlmerge_test.go

```diff
@@ -65,7 +65,7 @@ func TestMergeSDLs(t *testing.T) {
 
 const (
 	accountSchema = `
-		extend type Query {
```

Comment:
I believe it will be better not to modify old schema for this test, but add additional schema along this one and additional test case covering this change